src/R/Editor/Impl/Data/WorkspaceVariableProvider.cs

```diff
@@ -111,13 +115,42 @@ internal sealed class WorkspaceVariableProvider : RSessionChangeWatcher, IVariab
                                     .ToArray();
                         }
                     }
+                } else {
+                    // May be a package object line mtcars$
+                    variableName = TrimTrailingSelector(variableName);
+                    var sessionProvider = EditorShell.Current.ExportProvider.GetExportedValue<IRSessionProvider>();
+                    var session = sessionProvider.GetOrCreate(GuidList.InteractiveWindowRSessionGuid);
+                    var infoList = session.DescribeChildrenAsync("NULL", variableName, REvaluationResultProperties.HasChildrenProperty, 
```

Comment:
Use `REnvironments.GlobalEnv` (or whatever is most appropriate from that class) instead of "NULL". I'm actually kinda surprised this even works - looking at the source code of `describe_children` in `eval.R`, if it does, it's purely by accident.